src/Compilers/CSharp/Portable/Symbols/TypeSymbolWithAnnotations.cs

```diff
@@ -82,8 +148,10 @@ public static NullableAnnotation AsSpeakable(this NullableAnnotation annotation,
         /// </summary>
         public static NullableAnnotation JoinForFixingLowerBounds(this NullableAnnotation a, NullableAnnotation b)
```

Comment:
>JoinForFixingLowerBounds [](start = 41, length = 24)

nit: should this method still be called `Join` given that it doesn't pertain to states, and states and annotations are now separate things? (same question for `MeetForFixingUpperBounds`)

Reply:
I intend to rename all the meet and join methods in a separate PR when I simplify the annotations to have only speakable values.

---
In reply to: [260031348](https://github.com/dotnet/roslyn/pull/33648#discussion_r260031348) [](ancestors = 260031348)

Reply:
Yes, it should be called Join, as the values form a lattice which we use as a lattice.

---
In reply to: [260056842](https://github.com/dotnet/roslyn/pull/33648#discussion_r260056842) [](ancestors = 260056842,260031348)